src/app/components/LiveLabel/types.ts

```diff
@@ -3,6 +3,7 @@ export interface LiveLabelProps extends TextProps {
 }
 
 export interface TextProps {
+  className?: string;
```

Comment:
Not for this PR specifically, but I wonder should we globally extend `IntrinsicAttributes` to have `className` as an only property? Seems like a common enough pattern we have now for passing down `className` into components. Would just save having to type `className` all the time.

Reply:
This may not be generically possible though!

Reply:
Yeah if possible I think this is a good shout, as we have to pass `className` down to be able override default styles, which is the whole basis of this compound component pattern

Reply:
I can fire up a quick tech ticket for this

Reply:
Cheers 👍 Could be worth looking in to, unless it becomes too complicated to account for all flavours of JSX props. Hoping there was a generic way of doing it, but from a quick Google I couldn't see anything too obvious.